src/jit/gentree.h

```diff
@@ -857,6 +836,14 @@ struct GenTree
 
     //----------------------------------------------------------------
 
+#if defined(DEBUG)
+    #define GTF_DEBUG_MORPHED         0x00000001  // the node has been morphed (in the global morphing phase)
```

Comment:
Rename to `GTF_DEBUG_NODE_MORPHED` to signal the fact that it is node rather than tree specific and to make it consistent with `GTF_DEBUG_NODE_SMALL/LARGE`?